src/ServiceControl/ExternalIntegrations/ExternalIntegrationsHostBuilderExtensions.cs

```diff
@@ -4,7 +4,7 @@ namespace ServiceControl.ExternalIntegrations
     using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.Hosting;
 
-    static class ExternalIntegrationsHostBuilderExtensions
+    static class PersistenceHostBuilderExtensions
```

Comment:
Why did this change? Should the filename be changed to match?

Reply:
Must have been an auto rename change when I copied the file and renamed the copied version - I have changed it back.